### PR TITLE
Add new workflow named Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack

### PR DESCRIPTION
**Pull request by ncramer-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

workflow1

workflow2

#### Job

windowsdefenderatp

#### Artifact
slack1

### CODE REVIEW

**Comment by ncramer-r7** [comment]
```Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack
[*] Validating workflow at Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!
```

**Comment by ncramer-r7** [comment]
Screen Shot 2020-08-17 at 4 20 34 PM

File: workflows/Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack/help.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Description
+
+This workflow blacklists or unblacklists indicators with Microsoft Defender ATP via Slack command and reports information back to Slack.
+Indicators supported in this workflow are SHA1 hashes, SHA256 hashes, IPv4 addresses, IPv6 addresses, URLs and domains.
```
Comment (mrinehart-r7) [inline workflows/Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack/help.md:4]:
```suggestion
Indicators supported in this workflow are MD5 hashes, SHA1 hashes, SHA256 hashes, IPv4 addresses, IPv6 addresses, URLs and domains.
```

File: workflows/Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+extension: workflow
+products: ["insightconnect"]
+name: Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack
+title: Blacklist Indicators with Microsoft Defender ATP from Slack
+description: Use the simplicity of Slack to blacklist and unblacklist indicators such as SHA1 hashes, SHA256 hashes, domains, URLS and IP addresses with Microsoft Defender ATP.
```
Comment (mrinehart-r7) [inline workflows/Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack/workflow.spec.yaml:5]:
```suggestion
description: Use the simplicity of Slack to blacklist and unblacklist indicators such as hashes, domains, URLs, and IP addresses with Microsoft Defender ATP.
```
Just a bit more succinct so it fits better

**Pull request merged**